Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -27,6 +29,7 @@
 using Windows.UI.Input;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
```

Comment:
newly added System.Linq, System.Reflection, Windows.UI.Xaml.Automation.Provider are not needed.

Reply:
Have no idea how those got added since I certainly didn't add them myself... Perhaps some ReSharper magic..